test/spec/FindInFiles-test.js

```diff
@@ -785,6 +785,7 @@ define(function (require, exports, module) {
                 gotChange = false;
                 oldResults = null;
                 wasQuickChange = false;
+                FindInFiles.searchModel.clear();
```

Comment:
I don't have strong opinion here and I agree is it a bit odd.
My only doubt is `clearSearch` reset also `findOrReplaceInProgress`: I'm not sure how much it is important.